native-maven-plugin/src/main/java/org/graalvm/buildtools/maven/NativeBuildMojo.java

```diff
@@ -108,8 +110,8 @@ public class NativeBuildMojo extends AbstractNativeMojo {
     @Parameter(property = "classpath")
     private List<String> classpath;
 
-    @Parameter(property = "useArgFile", defaultValue = "true")
-    private boolean useArgFile;
+    @Parameter(property = "useArgFile")
+    private Boolean useArgFile;
```

Comment:
@alvarosanchez I'm not sure if this is the proper way to handle this. 
Per [PluginParameterExpressionEvaluator](https://maven.apache.org/ref/3.8.5/maven-core/apidocs/org/apache/maven/plugin/PluginParameterExpressionEvaluator.html) we could do this by binding this property to something that will be initialized on inject-time, but I don't know where would optimal place for that initialization code be?

Reply:
This would certainly work. However, I'd initialise the value (what is done in lines 164-166) in the constructor.

Reply:
Actually, now that I think about it, maybe the constructor is too early. I think Plexus will set via reflection the value of annotated fields after constructing the object, but I'm not really sure.

I prefer [constructor injection](https://github.com/micronaut-projects/micronaut-maven-plugin/blob/master/src/main/java/io/micronaut/build/DockerMojo.java#L52-L56) to avoid that.